frontend/src/routes/Governance/discovered/DiscoveredPolicies.tsx

```diff
@@ -12,6 +12,35 @@ import { ClusterPolicyViolationIcons2 } from '../components/ClusterPolicyViolati
 import { AcmTableWithEngine } from '../common/AcmTableWithEngine'
 import { uniq } from 'lodash'
 
+interface ISourceFilter {
+  label: string
+  value: string
+}
+
+export const getSourceFilter = (data: DiscoverdPolicyTableItem[]): ISourceFilter[] => {
+  const uniqMap: { [key: string]: ISourceType } = {}
+  data?.forEach((data: DiscoverdPolicyTableItem) => {
+    if (data.source?.type) {
+      const key = data.source.type + data.source.parentName + data.source.parentNs
```

Comment:
I'm being paranoid but this makes sure there is no overlap.
```suggestion
      const key = data.source.type + '_' + data.source.parentName + '_' + data.source.parentNs
```